<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChifoumiVue</class>
 <widget class="QMainWindow" name="ChifoumiVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chifoumi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="vAllChifoumi">
      <item>
       <layout class="QGridLayout" name="gridPartieHaute">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lJoueur">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Vous</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="scoreJoueur">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="coupMachine">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="ressourceChifoumi.qrc">:/chifoumi/images/rien_115.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="coupJoueur">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="ressourceChifoumi.qrc">:/chifoumi/images/rien_115.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lMachine">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Machine</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="scoreMachine">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="versus">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ressourceChifoumi.qrc">:/chifoumi/images/versus_70.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Gagnant Ã </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hPartieBasse">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vInterfaceJoueur">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="gbBoutonsCoups">
            <property name="minimumSize">
             <size>
              <width>509</width>
              <height>191</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100000</width>
              <height>191</height>
             </size>
            </property>
            <property name="title">
             <string>Choisissez une figure :</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pierre">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normalon>:/chifoumi/images/pierre_115.png</normalon>
                  </iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>127</width>
                   <height>128</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="papier">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normalon>:/chifoumi/images/papier_115.png</normalon>
                  </iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>127</width>
                   <height>128</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="ciseau">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normalon>:/chifoumi/images/ciseau_115.png</normalon>
                  </iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>127</width>
                   <height>128</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hBoutNewPartie">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="bNewPartie">
              <property name="text">
               <string>Jouer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="actionA_propos_de"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionA_propos_de">
   <property name="text">
    <string>&amp;A propos de ...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Alt+F3</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressourceChifoumi.qrc"/>
 </resources>
 <connections/>
</ui>
